vtex-apps/place-components
# CHK-468: Render provider logo

#### What problem is this solving?

Render logo for geolocation provider on LocationSearch component. Depends on vtex-apps/geolocation-graphql-interface/pull/5.

Also refactors:
- move `SuggestAddresses` query into a custom hook
- improve variable names for `GetAddress` query
- explicitly type hooks

#### How should this be manually tested?

- [Workspace](https://w0geo--checkoutio.myvtex.com/cart/add?sku=289)
- Go to shipping
- Start to type a valid address. You can use "Praia de b"
- Assert that the powered by Google logo is displayed on bottom right of the suggestion list

#### Checklist/Reminders


#### Screenshots or example usage

You should see this:

<img src="https://user-images.githubusercontent.com/26108090/100797919-a2732980-3401-11eb-9f10-f8be7d5c3635.png" width="450" />
